templates/tfengine/recipes/iam_members.hcl

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+schema = {
+  title                = "Recipe for iam bindings"
+  properties = {
+    iam_members = {
+      description = <<EOF
+        [Module](https://github.com/terraform-google-modules/terraform-google-iam)
+        
+        parent_id should be the following for each type of resource:
+        project: project IDs.
+        storage_bucket : Storage bucket names.
+        folder: folder IDs.
+        billing_account: billing_account IDs.
+        organization: organizations.
+      EOF
+      description = "[Module](https://github.com/terraform-google-modules/terraform-google-iam)"
+      type                 = "object"
+      additionalProperties = false
+      patternProperties = {
+        "^storage_bucket|project|organization|folder|billing_account$": {
+          type = "array"
+          items = {
+            type = "object"
+            required = [
+              "parent_ids",
+              "bindings"
+            ]
+            properties = {
+              # parent_id should be the following for each type of resource:
+              # project: project IDs.
+              # storage_bucket : Storage bucket names.
+              # folder: folder IDs.
+              # billing_account: billing_account IDs.
+              # organization: organizations.
+              parent_ids = {
+                description = "Ids of the parent to assign the bindings."
```

Comment:
You can put the comment above in the description here. It supports heredocs, like:
```
parent_ids = {
    description = <<EOF
        Ids of the parent to assign the bindings.

        Should be the following for each type of resource:
            project: project IDs.
            storage_bucket : Storage bucket names.
            folder: folder IDs.
            billing_account: billing_account IDs.
            organization: organizations.
    EOF
    ...
}
```